hack/quickstart/init-worker.sh

```diff
@@ -34,6 +34,7 @@ function init_worker_node() {
     # Setup kubeconfig
     mkdir -p /etc/kubernetes
     cp ${KUBECONFIG} /etc/kubernetes/kubeconfig
+    cp /home/core/assets/tls/ca.crt /etc/kubernetes/ca.crt
```

Comment:
Hmm thinking about this more though -- any instructions we provide are still going to have to say "copy over the CA cert", so not sure it's really much benefit to try and parse the kubeconfig in bash (and we're just obscuring the needed steps).

So maybe we just make sure to copy the ca.crt in this step (I had wrong link in previous comment): https://github.com/kubernetes-incubator/bootkube/blob/master/hack/quickstart/init-worker.sh#L54-L55